packages/react-sdk/src/db/style.ts

```diff
@@ -0,0 +1,49 @@
+import { z } from "zod";
+import type { StyleProperty, StyleValue } from "@webstudio-is/css-data";
+import { SharedStyleValue } from "@webstudio-is/css-data";
+import { type ComponentName, getComponentMeta } from "../components";
+
+export type PresetStylesItem = {
+  component: ComponentName;
+  property: StyleProperty;
+  value: StyleValue;
+};
+
+// @todo can't figure out how to make component and property to be enum
+export const zPresetStylesItem = z.object({
+  component: z.string(),
```

Comment:
Zod requires tuples with at least one element as an argument to `z.enum`:
```suggestion
  component: z.enum(getComponentNames() as [ComponentName]),
```

Reply:
cool, thanks
Will fix in separate PR